### PR TITLE
began css and created schema.sql

### PR DESCRIPTION
**Pull request by Gr8-Dayne**

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+CREATE TABLE IF NOT EXISTS
+vehicles (
+  ID SERIAL PRIMARY KEY,
+  search_Make VARCHAR(255) NOT NULL,
+  search_Model VARCHAR(255) NOT NULL,
+  search_Year NUMERIC (18,6),
+  search_Price INTEGER,
+  search_Milage FLOAT,
+  search_Model VARCHAR(255) NOT NULL,
```
Comment (phongvdoan) [inline schema.sql]: delete one search_Model

File: schema.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  Description TEXT,
```
Comment (phongvdoan) [inline schema.sql]: need to include image_URL

File: schema.sql [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  Foreign Key: Username_ID INTEGER
```
Comment (phongvdoan) [inline schema.sql]: need to include url for craigslist

File: schema.sql (hunk 2 above)
Comment (phongvdoan) [inline schema.sql]: do we need?

File: schema.sql (hunk 3 above)
Comment (Gr8-Dayne) [reply schema.sql]: included now, not in this pull request.

File: schema.sql (hunk 2 above)
Comment (Gr8-Dayne) [reply schema.sql]: included now, not in this pull request.

Comment (Gr8-Dayne) [reply schema.sql]: Negative.

File: schema.sql (hunk 1 above)
Comment (Gr8-Dayne) [reply schema.sql]: Done.

**Pull request merged**